libyul/optimiser/SSATransform.h

```diff
@@ -86,6 +92,10 @@ class SSATransform: public ASTModifier
 		m_nameDispenser(_nameDispenser), m_variablesToReplace(_variablesToReplace)
 	{ }
 
+	// Create a statement similar to "let a_i := a", which is insert right after the location
```

Comment:
I think the comment is too specific - the behaviour of the function is not related to control-flow joins.

Reply:
Naturally I feel like it functions like phi function, also the only case we want this statement is when control flow joins. Does this make sense?

Reply:
If you want to keep then, then please as "This function is usually used after control flow joins.